src/PIL/ImageStat.py

```diff
@@ -97,14 +97,14 @@ def _getmean(self):
 
     def _getmedian(self):
         """Get median pixel level for each layer"""
-
         v = []
-        for i in self.bands:
+        histogram_length = len(self.h)
+        for i, band in enumerate(self.bands):
             s = 0
             half = self.count[i] // 2
```

Comment:
This `i` should have been changed to `band`.

Reply:
But the CI still passed, so perhaps this PR has helped identify a gap in testing?

Reply:
`i` and `band` are the same, as `self.bands` is just a range, so there's no functional difference
https://github.com/python-pillow/Pillow/blob/a4e5dc219c5185e4a655327c7ede38846dab189b/src/PIL/ImageStat.py#L40